File: ImagePickerMaster/src/main/java/com/example/apple/glidetest/bean/Change.java
package com.example.apple.glidetest.bean;

/**
 * Created by Apple on 17/7/17.
 */

public class Change {

    public static final int ADD = 0;
    public static final int REMOVE = 1;
    private int type;
    public String path;

    public Change(int type, String path) {
        this.type = type;
        this.path = path;
    }

    public boolean isAdd() {
        return type == ADD;
    }

//    public int getAllIndex() {
//        List<String> imgs = FolderProvider.getInstance().getSelectedFolder().imgs;
//        return ListUtils.getIndexInList(imgs, path)+1;
//    }
}

File: ImagePickerMaster/src/main/java/com/example/apple/glidetest/bean/Change.kt
package com.example.apple.glidetest.bean

/**
 * Created by Apple on 17/7/17.
 */

class Change(private val type: Int, var path: String) {

    val isAdd: Boolean
        get() = type == ADD

    companion object {
        val ADD = 0
        val REMOVE = 1
    }

}
